metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -973,6 +1043,26 @@ def add_table_to_dataset_container(
             dataset_urn=dataset_urn,
         )
 
+    def add_model_to_schema_container(
+        self, model_urn: str, schema: Schema
+    ) -> Iterable[MetadataWorkUnit]:
+        schema_container_key = self.gen_schema_key(schema)
+        yield from add_entity_to_container(
+            container_key=schema_container_key,
+            entity_type="mlModelGroup",
```

Comment:
Container types should be constants.

Reply:
updated - we did the similar thing in [mlflow connector](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/mlflow.py#L738), but i noticed that we keep separate `constants.py` files with other connectors. let me know which is the best approach.